controller/src/beerocks/master/db/db.cpp

```diff
@@ -1232,6 +1232,48 @@ const std::vector<beerocks_message::sWifiChannel> db::get_hostap_supported_chann
     return n->hostap->supported_channels;
 }
 
+/**
+ * @brief Add supported operating class to the database.
+ * Currently this function is a wrapper which converts the operating
+ * class to a set of supported channels and updates the list of currently
+ * supported channels.
+ * 
+ * @param mac radio mac
```

Comment:
In EasyMesh terminology, it would be Radio UID. (nitpcik)

Reply:
See `handle_non_intel_slave_join` - this is the RUID. We keep the same convention of calling it radio mac only for alignment with the current code base, but @arnout 's comment is correct so I'll add it to the description.